### ISSUE TITLE
Replace the old Audacity theme

### ISSUE THREAD

**Issue by CHJ85** [tenacityteam/tenacity-legacy#29]
With this new fork of Audacity, obviously the default themes need to be replaced.
I found this neat project here on github https://github.com/visoart/audacity-themes
These are some really good themes. Much better than the default ones in my opinion.
That being said, I do think that longer down the line, the entire theme engine need to be modernized.
It's like, do we even want it to look like audacity anymore?
This is a great opportunity to redo the entire look and feel to it.

**Comment by ghost** [comment]
I think Audacity should be yellow and red, like sneed, and like the emojis under your comment

**Comment by l0go** [comment]
[This](https://www.deviantart.com/higorsm25/art/Audacity-for-Windows-10-Concept-553216132) is one of my personal favorite concepts for an audacity theme. Although it needs more labeling and the top bar should be cleaned up.

**Comment by rolandixor** [comment]
I would suggest keeping it simple to start, seeing as it would probably be wise to move from wxWidgets to another engine.
Reusing an existing theme would probably be the wisest move, IMHO.

**Comment by HumbleDeer** [comment]
> I would suggest keeping it simple to start, seeing as it would probably be wise to move from wxWidgets to another engine.
> Reusing an existing theme would probably be the wisest move, IMHO.

Why move away from wxWidgets though?

**Comment by caughtquick** [comment]
> > I would suggest keeping it simple to start, seeing as it would probably be wise to move from wxWidgets to another engine.
> 
> > Reusing an existing theme would probably be the wisest move, IMHO.
> 
> 
> 
> Why move away from wxWidgets though? 

It's outdated and limiting compared to more modern options like qt. Audacity/tenacity also require a vendored fork of wxWidgets which makes it a challenge to package (why arch and void are on 2.x.x versions)

**Comment by Semisol** [comment]
Also some concerns have been voiced about switching to a new theme, so I would like community feedback first.

**Comment by falkTX** [comment]
Let's please stick with what upstream has right now.
I got the impression they might move to WxQt backend at some point, it is not clear yet.

**Comment by caughtquick** [comment]
> Let's please stick with what upstream has right now.
> 
> I got the impression they might move to WxQt backend at some point, it is not clear yet.
> 
> 

I agree, the project first needs to get off the ground as a suitable audacity fork that meets the main goals of being a "demused-audacity". Anything more than that is too early to be decided IMO.

**Comment by CHJ85** [comment]
@caughtquick Fair enough. But in order to get off the ground, it needs a facelift. Something that pops out and gives people the impression that it's not just Audacity without telemetry. Because at the end of the day, people don't really care that much about telemetry. They make a big fuzz about it right now because that's how the internet works. But in about a month, they'll move on to the next thing. My point is. Whether or not this fork stays relevant depends on what big changes are being made right now that all eyes are looking.

**Comment by falkTX** [comment]
some of us dont want that.
I want an audacity that is spyware/tracking free, not something that behaves or looks overall different.

and btw, the "facelift" is already in upstream audacity's plans.

**Comment by CHJ85** [comment]
@falkTX Same. But let's face it. Most people are all about the hype. So in order for this project to stay relevant and active, there needs to be more to it than being anti-spyware. Things like a better UI, more features and so on.

**Comment by falkTX** [comment]
Hype is what marketing people use to sell products.
We need to invest on the long-term sustainability of the project, not rush things to get attention.

Audacity has built a reputation over the years. A good fork will do the same.

**Comment by CHJ85** [comment]
@falkTX Yes. That is absolutely true. I'd still make a unique gui though. Who knows what the Muse Group decide to slap on top of Audacity? If they're going for a look similar to Ultimate Guitar or MuseScore in order to fit in with their family of apps, it's worth keeping in mind that the gui might not even be open source.

**Comment by falkTX** [comment]
Audacity will remain opensource, it is a promise by them.
Their upcoming mobile app of audacity will not be, the same thing already happened with musescore, so we have a precedent.
This fork is about the desktop application, not anything else.

**Comment by CHJ85** [comment]
@falkTX Right. And you know what else they promised? No telemetry. I don't trust them. Having a plan b doesn't hurt.

**Comment by falkTX** [comment]
Show me where they promised no telemetry.
IIRC they said they would back off and not merge the big messy PR, which they did.

I also don't trust them, but it seemed obvious to me that telemetry was/is going in one way or the other. At some point Tantacrul mentioned that it is a requirement by his employer.

**Comment by CHJ85** [comment]
@falkTX They promised it by rejecting the pull request after all the backlash. Then they went complete 180 a few weeks later. And now, here we are.

**Comment by falkTX** [comment]
huh? they didnt go back to their promise of merging that specific PR, or using google+yandex (so far at least).
It was not a 180, you were misled if you think their intentions were to abandon analytics.
Audacity under Muse Group will have analytics no matter what.

**Comment by caughtquick** [comment]
@akleja is attempting this in #315.

**Comment by ghost** [comment]
SNEED THEME
